TicTacToe.hpp

```diff
@@ -22,16 +22,36 @@ class TicTacToe
 		bool add(int x, int y); // Anushka
 		bool remove(int x, int y); // Anushka
 
-		int getActionsLeft(); // Minh
-		int getTeamSide(); // Minh
+		int getActionsLeft() {
+			return actionsLeft;
+			//I think that the actionsLeft should be changed when the functions add and remove are called
+		} // Minh
+		int getTeamSide() {
+			//the teamSide should be changed using the changeTurn function. I can't really do anything here
+			return teamSide;
+		} // Minh
 		int getVictory() {
 			return victory;
 		}
 
 	private:
 		// These functions should only be called from within this class.
 		void changeTurn(); //Phillip
-		void checkVictory();
+		void checkVictory() {
+			int blue = 1,red = -1,blue_points,red_points;
+			blue_points = checkVictoryHorizontal(blue) + checkVictoryDiagonal(blue) + checkVictoryVertical(blue);
+			red_points = checkVictoryHorizontal(red) + checkVictoryDiagonal(red) + checkVictoryVertical(red);
+			if(blue_points == 4) {
+				victory = 1;
+			}
+			else if(red_points = 4) {
+				victory = -1;
+			}
+			else {
+				blue > red ? victory = 1 : victory = -1;
```

Comment:
We don't need this case, because we only care if someone has won, not if they are in the lead.

Reply:
ok